### PR TITLE
Update API review process image (corefx->runtime)

### PR DESCRIPTION
**Pull request by carlossanlop**
Change corefx for runtime in the image. Also fix a small typo in the document text.

[Preview](https://github.com/carlossanlop/runtime/blob/ebeb418926e0729d43d6f3b4c5b33d68630c3868/docs/project/api-review-process.md) - Note: GitHub did not show the updated image the first time I saw it. I had to refresh the page a couple of times so the new image would get shown.

Edit: Ignore the Dotnet-GitSync-Bot actions. I accidentally pushed some unrelated files the first time.

### CODE REVIEW

**Review by ViktorHofer — APPROVED** [review]

**Comment by stephentoub** [comment]
@maryamariyan, any idea why the bot added the "new-api-needs-documentation" here?  Thanks.

**Comment by stephentoub** [comment]
Oh, nevermind, I see Carlos' comment: "Ignore the Dotnet-GitSync-Bot actions. I accidentally pushed some unrelated files the first time."

File: docs/project/api-review-process.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ APIs and some code samples that show how it should be used. If changes are neces
 
 4. **Owner makes decision**. When the owner believes enough information is available to make a decision, they will update the issue accordingly:
 
-    * **Mark for review**. If the owner believes the proposal is actionable, they will label the issue with `api-ready-for-review`. [Here is a good example](https://github.com/dotnet/corefx/issues/4547) of as strong API proposal.
+    * **Mark for review**. If the owner believes the proposal is actionable, they will label the issue with `api-ready-for-review`. [Here is a good example](https://github.com/dotnet/corefx/issues/4547) of a strong API proposal.
```
Comment (stephentoub) [inline docs/project/api-review-process.md]: As long as we're changing this, shouldn't it be a link to a dotnet/runtime issue?

**Comment by stephentoub** [comment]
As long as we're updating this, is there some other image format that would be easier to update, like a .svg?

**Review by stephentoub — APPROVED** [review]

**Comment by carlossanlop** [comment]
f we can get the original template, which seems like it was created with an Office app, we can quickly save the image. @ViktorHofer may still have the template.

**Pull request merged**